docs/start-application.md

```diff
@@ -0,0 +1,29 @@
+# Sparkle Web App
+
+Codebase for the Sparkle platform, brought to you by [Sparkle](https://sparklespaces.com/), a platform for the most immersive & interactive virtual events.
+
+<!-- section 1 -->
+
+### Launch application
+
+You're going to need 2 terminal tabs to launch Sparkle application locally. Make sure that you're located in the application root folder and follow steps below:
+* Run `npm run firebase:emulate-functions` in the first tab
+* Run `npm run start` in the second
+
+Once the launch is complete you may proceed to http://localhost:3000/v/bootstrap
+
+You can register a new user by pressing Log In in the top right corner:
+<!-- ![image](./images/creating-new-environments/59-create-account.png) -->
+
+Once redirected to authorization form select Create account:
+* Fill in necessary data and press Continue
+<!-- ![image](./images/creating-new-environments/60-register-form.png) -->
+
+* Fill in your Username and upload custom avatar, or select from the default ones. Then press Create my profile:
+<!-- ![image](./images/creating-new-environments/61-user-form.png) -->
+
+Navigate to http://localhost:3000/v/bootstrap where you can enter your first venue.
+
+Now you may also visit Admin dashboard to edit venues, create new ones or manage users.
+* https://example.sparkle.space/admin
+* https://example.sparkle.space/admin_v2
```

Comment:
This probably needs similar tweaks as per my other review comments here.

I wonder if we should call this section 'getting started' and combine it with some of the other 'local dev' things.

Eg.

```markdown
## Getting Started

### First time setup

If you don't already have an environment... [link to create firebase docs]

Things to do to set up the project locally...

### Running the application

More instructions here...
```

It's worth looking at an older version of the README on the main repo, before any of these changes were made, and before the 'using emulators' section was added. I expect that's going to be far closer to what we want. If not already exactly what we want.

Reply:
fixed and added missing information in f6618f8